### PR TITLE
[web3-wrapper] start jsonRpcRequestId at 1, not 0

### PR DESCRIPTION
**Pull request by WyseNynja**
I'm trying to use the 0x-watcher against a parity websocket rpc and getting an error that I think is the same as https://github.com/ethereum/web3.js/issues/1967

## Description

Without this change, I'm getting this error:

```
TypeError: Cannot read property 'id' of undefined
    at WebsocketProvider._addResponseCallback (/usr/src/myapp/node_modules/web3-providers-ws/src/index.js:222:39)
    at WebsocketProvider.send (/usr/src/myapp/node_modules/web3-providers-ws/src/index.js:281:10)
    at /usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:78:36
    at new Promise (<anonymous>)
    at Object.<anonymous> (/usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:74:39)
    at step (/usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:32:23)
    at Object.next (/usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:13:53)
    at /usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:3:12)
    at promisifiedFunction (/usr/src/myapp/node_modules/@0x/utils/lib/src/promisify.js:72:16)
    at Web3Wrapper.<anonymous> (/usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:932:74)
    at step (/usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:43:23)
    at Object.next (/usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:24:53)
    at /usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:18:71
    at new Promise (<anonymous>)
    at __awaiter (/usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:14:12)
    at Web3Wrapper.sendRawPayloadAsync (/usr/src/myapp/node_modules/@0x/web3-wrapper/lib/src/web3_wrapper.js:925:16)
    at EventWatcher.<anonymous> (/usr/src/myapp/node_modules/@0x/order-watcher/lib/src/order_watcher/event_watcher.js:131:64)
    at step (/usr/src/myapp/node_modules/@0x/order-watcher/lib/src/order_watcher/event_watcher.js:43:23)
    at Object.next (/usr/src/myapp/node_modules/@0x/order-watcher/lib/src/order_watcher/event_watcher.js:24:53)
    at /usr/src/myapp/node_modules/@0x/order-watcher/lib/src/order_watcher/event_watcher.js:18:71
```

I doubt anyone is depending on the initial id being 0. I'm testing this on my app now. It might not be the right fix at all.

## Testing instructions

Work in progress.

```
git clone https://github.com/0xProject/0x-monorepo.git
cd 0x-monorepo/packages/web3-wrapper
git fetch origin pull/1227/head:pull1227
git checkout pull1227
yarn link
```

## Types of changes

* Bug fix (non-breaking change which fixes an issue)

## Checklist:

*   [x] Prefix PR title with `[WIP]` if necessary.
*   [x] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [x] Add tests to cover changes as needed. (none needed)
*   [x] Update documentation as needed.
*   [x] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Comment by WyseNynja** [comment]
I'm having trouble testing this. I don't write very much node and `yarn link` is not working how I expect.

**Comment by LogvinovLeon** [comment]
Thanks for the contribution

**Comment by fabioberger** [comment]
@WyseNynja can you add a note in the `CHANGELOG.json` file for `web3-wrappers` about this change? It should be a patch bump, since it's a bug fix.

**Comment by WyseNynja** [comment]
I think I did that right. Let me know.

File: packages/order-watcher/CHANGELOG.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "2.2.2",
+        "changes": [
+            {
+                "note": "start jsonRpcRequestId at 1, not 0",
+                "pr": 1227
+            }
+        ],
+        "timestamp": 1541977777
```
Comment (fabioberger) [inline packages/order-watcher/CHANGELOG.json]:
```suggestion
```

The timestamp get's added when published

File: packages/order-watcher/CHANGELOG.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "2.2.2",
+        "changes": [
+            {
+                "note": "start jsonRpcRequestId at 1, not 0",
```
Comment (fabioberger) [inline packages/order-watcher/CHANGELOG.json]: Mind adding a bit more of "why"? Is there a PR/issue we could link to?

Comment (LogvinovLeon) [reply packages/order-watcher/CHANGELOG.json]:
```suggestion
                "note": "Start jsonRpcRequestId at 1, not 0 as 0 breaks Parity websocket RPC",
```

**Comment by WyseNynja** [comment]
Thanks everyone! This should be ready now.

**Comment by fabioberger** [comment]
Thanks for the PR @WyseNynja!

**Pull request merged**